index.html

```diff
@@ -164,36 +223,33 @@ <h3 class="p-matrix__title">Small</h3>
     <section class="p-strip is-bordered" id="quick-start">
       <div class="row">
         <h2>Quick start</h2>
-        <p>To install the <a href="https://snapcraft.io/microk8s">MicroK8s&nbsp;snap</a>:</p>
+        <p>We recommend you use Ubuntu. On Mac or Windows, you can use Multipass to give yourself Ubuntu VMs on demand. To install the <a href="https://snapcraft.io/microk8s">MicroK8s&nbsp;snap</a>:</p>
       </div>
       <div class="row u-equal-height">
         <div class="col-8">
           <div class="p-code-copyable">
             <input class="p-code-copyable__input" aria-label="sudo snap install microk8s --classic" value="sudo snap install microk8s --classic" readonly="readonly">
             <button class="p-code-copyable__action">Copy to clipboard</button>
           </div>
-          <p>Don’t have the <code>snap</code> command? <a href="https://snapcraft.io/docs/core/install" class="p-link--external">Install <code>snapd</code> first.</a></p>
-        </div>
-      </div>
-      <div class="row">
-        <div class="col-8">
-          <hr>
-          <h4>For the current published versions</h4>
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" aria-label="snap info microk8s" value="snap info microk8s" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
-          <p>It is possible to select a specific version like 1.11 and get only stable or RC updates to that version. By default, you will get the current major version with upgrades when new major versions are published.</p>
+          <p>
+            <code>snap info microK8s</code> shows the current published k8s versions. It is possible to select a specific version like 1.11 and get only stable or RC updates to that version. By default, you will get the current major version with upgrades when new major versions are published.
```

Comment:
Consider adding some text to the start of this paragraph before the code. For example: "Running the command <code>snap info microK8s</code> shows the current published..."